Formula/kopia.rb

```diff
@@ -0,0 +1,48 @@
+class Kopia < Formula
+  desc "Fast and secure open-source backup"
+  homepage "https://kopia.io"
+  url "https://github.com/kopia/kopia.git",
+      tag:      "v0.10.4",
+      revision: "3447f1865f0b0536653aa6e9c07896ce73c3e600"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X github.com/kopia/kopia/repo.BuildInfo=#{Utils.git_head}
+      -X github.com/kopia/kopia/repo.BuildVersion=#{version}
+      -X github.com/kopia/kopia/repo.BuildGitHubRepo=kopia/kopia
```

Comment:
Should this reference upstream or whoever build it? If it's the latter this should be 
```suggestion
      -X github.com/kopia/kopia/repo.BuildGitHubRepo=#{tap.user}
```

Reply:
This is used by kopia to show upgrade notice when a new version is available on GitHub. Since only kopia/kopia is producing official builds and this formula is for that, hard coding is fine. 

We can make it an empty string to disable upgrade checks, perhaps, but I would prefer to keep it.

Reply:
Let's disable the upgrade checks. Formulae here are updated relatively quickly, but we still want to avoid the situation where a new tag has been cut at `kopia/kopia` but hasn't made it here, so a user sees that an upgrade is available but isn't able to do `brew upgrade` to get it.

Reply:
done